File: POO/guia02/Notas.java
public class Notas {

	private String nome;
	private float nota1;

    public Notas(String nome, float nota1) {
		this.nome = nome;
		this.nota1 = nota1;
    }

	public void setNome(String nome) {
    	this.nome = nome;
	}

	public String getNome() {
    	return nome;
	}

	public void setNota1(float nota1) {
    	this.nota1 = nota1;
	}

	public float getNota1() {
    	return nota1;
	}

}

File: POO/guia02/solution/notas.cpp
#include <string>

using namespace std;

class notas {
  string Nome;
  int Nota1;
  int Nota2;

public:
  notas(string nome, int nota1, int nota2)
      : Nome(nome), Nota1(nota1), Nota2(nota2){};
  /* virtual ~notas(); */
  int get() { return Nota1; }
};
int main() { notas Aluno = notas("danilo", 5, 6); }
